supernode/daemon/mgr/ha_mgr.go

```diff
@@ -0,0 +1,19 @@
+package mgr
+
+//HaMgr is the interface to implement supernode Ha.
+type HaMgr interface {
+	//ElectDaemonthe is the daemon progress to implement active/standby switch.
+	ElectDaemon(change chan int)
+
+	//HagetSupernodeState get supernode's status.
```

Comment:
s/get/gets. The third person singular should be used here. So do with the other comments.